local-cli/runIOS/runIOS.js

```diff
@@ -132,15 +135,21 @@ function runOnDevice(selectedDevice, scheme, xcodeProject){
   });
 }
 
-function buildProject(xcodeProject, udid, scheme) {
+function buildProject(xcodeProject, udid, scheme, configuration) {
   return new Promise((resolve,reject) =>
   {
-     const xcodebuildArgs = [
+     var xcodebuildArgs = [
       xcodeProject.isWorkspace ? '-workspace' : '-project', xcodeProject.name,
       '-scheme', scheme,
       '-destination', `id=${udid}`,
       '-derivedDataPath', 'build',
     ];
+    if (configuration) {
+      xcodebuildArgs = [
+        ...xcodebuildArgs,
+        '-configuration', configuration,
+      ];
+    }
```

Comment:
can you change this to push to `xcodebuildArgs` instead of creating a new array? That way you can keep using `const` above too :)